src/redux/configureStore.js

```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import booksReducer from './books/book';
 
 const store = configureStore({
-  reducer: {
-  },
+  reducer: { books: booksReducer },
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
```

Comment:
- Good job setting the configure store 😃

Reply:
Thank you